examples/complex-storage-structures/README.md

```diff
@@ -0,0 +1,340 @@
+# Storage refactoring
+
+In ink! v4.0.0-beta the way storage works was refactored.
+
+## Pre-ink! v4.0.0-beta storage
```

Comment:
Can you remove the section on how it was done before?

It was fine when you created the PR, but we have the migration guide from 3.x to 4.x by now, which covers it.